SpaceStoryComponents.py

```diff
@@ -9,31 +12,23 @@
 
-# Method for creating paragraph two, takes child name as input
+# Method for creating paragraph two, takes child name and pronouns as inputs
 # There are two possible story options, randomly selected using the random module
 # Also uses the random module to randomly select alien's appearance and sound from a list of options,
 # and to randomly select a spaceship part from a list of options for story option 1
 # Returns a string containing paragraph two of the story
-def paragraph_two(child_name):
+def paragraph_two(child_name, child_pronouns):
     rand_spaceship_part = random.choice(["doors", "door frames", "vents", "door locks"])
     rand_noise = random.choice(["a warbling wail", "a musical chime-like song", "a noise like a bubbling brook", "a trilling whistle"])
     rand_colour = random.choice(["hot pink", "deep purple", "lime green", "neon orange", "sunflower yellow"])
     rand_pattern = random.choice(["holographic stripes", "bumpy scales", "super fluffy fur", "a leathery hide", "silky downy hair","black polka dots", "white polka dots"])
-    alien_desc = f"{rand_colour} with {rand_pattern}"
+    alien_desc = f'{rand_colour} with {rand_pattern}'
 
-    option_one = f'''An unidentified lifeform has got stuck in the air lock of the ship - they need someone to come get it from the outside.
-    {child_name} bravely pulled on their space suit and jumped in a rocket.
-    In space, {child_name} parked their ship next to the ISS and went on a spacewalk to investigate the airlock.
-    The outer door was jammed open, and when {child_name} peeked inside they saw a cute little {alien_desc}. One of the tentacles of the alien was trapped in one of the {rand_spaceship_part} and it was crying.
-    Luckily {child_name} knew just what to do, and brought out a pack of space butter. After rubbing the tentacle in butter it slipped straight out! The alien cuddled up to {child_name} and then ate all the butter.'''
+    option_one = f"An unidentified lifeform has got stuck in the airlock of the ship - they need someone to come get it from the outside. {child_name} bravely pulled on {pronouns[child_pronouns]['possessive']} space suit and jumped in a rocket. In space, {child_name} parked {pronouns[child_pronouns]['possessive']} ship next to the ISS and went on a spacewalk to investigate the airlock. The outer door was wide open, and when {child_name} peeked inside {pronouns[child_pronouns]['subject']} saw a cute little {alien_desc}. One of the tentacles of the alien was trapped in one of the {rand_spaceship_part} and it was crying. Luckily {child_name} knew just what to do, and brought out a pack of space butter. After rubbing the tentacle in butter it slipped straight out! The alien cuddled up to {child_name} and then ate all the butter."
```

Comment:
Should line 29 read "cute little {alien desc} alien." rather than "cute little {alien desc}." ?